libs/platform/user/kernel_um.cpp

```diff
@@ -547,6 +557,10 @@ RtlLengthSid(_In_ PSID Sid)
 NTSTATUS
 RtlAddAccessAllowedAce(_Inout_ PACL Acl, _In_ unsigned long AceRevision, _In_ ACCESS_MASK AccessMask, _In_ PSID Sid)
 {
+    if (ebpf_fault_injection_inject_fault()) {
+        return STATUS_BUFFER_TOO_SMALL;
```

Comment:
Choose a different status code from msdn.

Reply:
As per the NTSTATUS in kernel_um.h, this was the closest.